### PR TITLE
Gce multimaster

### PR DESCRIPTION
**Pull request by stgleb**
Fix GCE High-Available cluster topology

- [ x ] Fix (a **bug** or other **issue** was fixed)
- [ ] Feature (**new functionality** was added)
- [ ] Breaking (**existing functionality** was affected)
- [ ] Other (please **explain** below)

Fixes: #1391 

- [ x ] [Documentation](https://supergiant.readme.io/v2.1.0-alpha/) was **needed**, and I updated it.
- [ ] [Documentation](https://supergiant.readme.io/v2.1.0-alpha/) was **not** needed.
- [ ] I understand [the Contribution Guidelines](https://supergiant.readme.io/docs/guidelines).

### CODE REVIEW

File: pkg/workflows/steps/kubeadm/kubeadm.go [hunk 1]
```diff
@@ -49,8 +49,16 @@ func (t *Step) Run(ctx context.Context, out io.Writer, config *steps.Config) err
 	config.KubeadmConfig.InternalDNSName = config.InternalDNSName
 	config.KubeadmConfig.ExternalDNSName = config.ExternalDNSName
 	config.KubeadmConfig.Token = config.BootstrapToken
-	config.KubeadmConfig.PrivateIP = config.Node.PrivateIp
+	config.KubeadmConfig.NodeIp = config.Node.PrivateIp
 	config.KubeadmConfig.AdvertiseAddress = config.Node.PrivateIp
+	config.KubeadmConfig.NodeIp = config.Node.PrivateIp
```
Comment (saheienko) [inline pkg/workflows/steps/kubeadm/kubeadm.go]: `NodeIp` has been already set.

File: templates/kubeadm.sh.tpl [hunk 2]
```diff
@@ -53,12 +52,12 @@ apiServer:
...
   extraArgs:
     authorization-mode: Node,RBAC
-    bind-address: {{ .PrivateIP }}
+    bind-address: {{ .PrivateIp }}
     {{ if .Provider }}cloud-provider: {{ .Provider }}{{ end }}
   timeoutForControlPlane: 8m0s
 controllerManager:
   extraArgs:
-    {{ if .Provider }}cloud-provider: {{ .Provider }}{{ end }}
+     {{ if .Provider }}cloud-provider: {{ .Provider }}{{ end }}
```
Comment (saheienko) [inline templates/kubeadm.sh.tpl]: Is it a required change?

Comment (stgleb) [reply templates/kubeadm.sh.tpl]: removed

**Comment by eldridgeh** [comment]
@stgleb I just tried to create a 3 master 3 node cluster in GCE and the third master failed on the third master at the kubeadm step.
```
[kubelet-start] Downloading configuration for the kubelet from the "kubelet-config-1.14" ConfigMap in the kube-system namespace
[kubelet-start] Writing kubelet configuration to file "/var/lib/kubelet/config.yaml"
[kubelet-start] Writing kubelet environment file with flags to file "/var/lib/kubelet/kubeadm-flags.env"
[kubelet-start] Activating the kubelet service
[kubelet-start] Waiting for the kubelet to perform the TLS Bootstrap...
error execution phase control-plane-join/etcd: error creating local etcd static pod manifest file: etcdserver: unhealthy cluster
time="2019-05-14T19:47:23+03:00" level=info msg="[kubeadm] - failed: kubeadm step: Process exited with status 1"
```

**Pull request merged**